consensus/polybft/signer/utils_test.go

```diff
@@ -1,99 +0,0 @@
-package bls
-
-import (
-	"encoding/hex"
-	"testing"
-
-	"github.com/0xPolygon/polygon-edge/types"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-)
-
-func Test_SingleSign(t *testing.T) {
-	t.Parallel()
-
-	validTestMsg, invalidTestMsg := testGenRandomBytes(t, messageSize), testGenRandomBytes(t, messageSize)
-
-	blsKey, err := GenerateBlsKey() // structure which holds private/public key pair
-	require.NoError(t, err)
-
-	// Sign valid message
-	signature, err := blsKey.Sign(validTestMsg, DomainValidatorSet)
-	require.NoError(t, err)
-
-	isOk := signature.Verify(blsKey.PublicKey(), validTestMsg, DomainValidatorSet)
-	assert.True(t, isOk)
-
-	// Verify if invalid message is signed with correct private key. Only use public key for the verification
-	// this should fail => isOk = false
-	isOk = signature.Verify(blsKey.PublicKey(), invalidTestMsg, DomainValidatorSet)
-	assert.False(t, isOk)
-}
-
-func Test_AggregatedSign(t *testing.T) {
```

Comment:
Maybe we should move `Test_AggregatedSign` and `Test_MakeKOSKSignature` to the new file also?

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1981/commits/98fca7d269902d4f595e5df69ae83c42c199e8dc